integration/spark-common/src/main/scala/org/apache/spark/sql/catalyst/CarbonDDLSqlParser.scala

```diff
@@ -720,18 +720,18 @@ abstract class CarbonDDLSqlParser extends AbstractCarbonSparkSQLParser {
         tableProperties.get(CarbonCommonConstants.DICTIONARY_EXCLUDE).get.split(',').map(_.trim)
       dictExcludeCols
         .foreach { dictExcludeCol =>
+          if (dictExcludeCol.contains(".") || dictExcludeCol.contains("[")) {
```

Comment:
do not check, for contains ".", get from field whether it is child column, then you can check whether that is given in include or exclude and throw exception